tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md

```diff
@@ -481,3 +481,6 @@ AccuracyChecker supports following set of adapters:
   * `output_name` - name of output node to convert.
 * `mask_to_binary_classification` - converts output of model represented as segmentation mask to `ArgMaxClassificationPrediction`. Class label calculated as comparision maximal probability in mask with given threshold.
   * `threshold` - probability threshold for label 1 (Optional, default 0.5).
+* `ssd_vgg16` - converting output of VGG based SSD model to `DetectionPrediction` representation.
```

Comment:
not sure that it is connveted with vgg16 backbone, this repo provide also other ssd implementation, possibly we need give more general name and more description why these adapter should be used

Reply:
Done